master/pkg/model/searcher_config.go

```diff
@@ -23,7 +23,7 @@ type SearcherConfig struct {
 	SingleConfig         *SingleConfig         `union:"name,single" json:"-"`
 	RandomConfig         *RandomConfig         `union:"name,random" json:"-"`
 	GridConfig           *GridConfig           `union:"name,grid" json:"-"`
-	AsyncHalvingConfig   *AsyncHalvingConfig   `union:"name,async_halving" json:"-"`
+	SyncHalvingConfig    *SyncHalvingConfig    `union:"name,sync_halving" json:"-"`
```

Comment:
thought: I guess I should point out that this is technically a user-facing, non-backward-compatible change, but the name is not actually documented, so it should probably be fine.

Reply:
note: i didn't find any experiment configs using this searcher in our examples.